### PR TITLE
updated the issue template markdown with a bit more information

### PR DESCRIPTION
**Pull request by jmhessel**
Updated ISSUE_TEMPLATE.md to potentially help cut down on the github issues that are just implementation questions. I don't know if it will help, but feel free to merge if this is interesting!

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
-Please make sure that the boxes below are checked before you submit your issue. Thank you!
+Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question rather than a bug in Keras, please consider asking your question on the [Google Group](https://groups.google.com/forum/#!forum/keras-users) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) rather than filing a GitHub issue.
```
Comment (fchollet) [inline ISSUE_TEMPLATE.md]: Actually, it's probably better to direct people to StackOverflow. Few people answer questions in the Google group. On SO and Slack more people are active and answering questions.

Comment (jmhessel) [inline ISSUE_TEMPLATE.md]: Thanks! Updating now.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question rather than a bug in Keras, please consider asking your question on the [Google Group](https://groups.google.com/forum/#!forum/keras-users) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) rather than filing a GitHub issue.
+Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question rather than a bug in Keras, instead of filing a GitHub issue, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras), consider [joining the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/), or check out the [Keras Google Group](https://groups.google.com/forum/#!forum/keras-users).
```
Comment (fchollet) [inline ISSUE_TEMPLATE.md]: Too fragmented -- I think we can remove the mention to the Google group. It's getting used less and less.

**Comment by fchollet** [comment]
LGTM

**Pull request merged**